kube-eagle/templates/deployment.yaml

```diff
@@ -31,6 +31,9 @@ spec:
 {{ toYaml .Values.podSecurityContext | indent 8 }}
       {{- end }}
       serviceAccountName: {{ .Values.serviceAccountName }}
+{{- if .Values.priorityClassName }}
+      priorityClassName: {{ .Values.priorityClassName }}
```

Comment:
I believe it must/should be quoted. What do you think about this?

```
{{ .Values.priorityClassName | quote }}
```

Reply:
quotes are not required https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#pod-priority